src/services/schedulingService.ts

```diff
@@ -130,6 +131,18 @@ class SchedulingService {
     }
   }
 
+  private async cancelEventWithoutEnoughInvites() {
+    const events = await this.eventService.getAllEvents();
+    const maxParticipants = (await this.configRepository.getConfig()).participants;
+
+    for (const event of events) {
+      if (event.accepted.length + event.invites.length < maxParticipants) {
+        this.planningService.removeSingleEvent(event);
+        await this.slackService.sendFailedEventNotification(event);
```

Comment:
Since we now may have individuals with open invites, should we expand this function to also send individual messages to them?